A3A/addons/core/functions/Supports/fn_initSupports.sqf

```diff
@@ -48,9 +48,9 @@ private _initData = [
     ["CARPETBOMBS",     "AREA", 0.5, 0.1, 200,   0, "u", ""],                            // balanced against airstrikes
     ["GUNSHIP",         "AREA", 0.2, 0.1, 0, 80, "", "vehiclesPlanesGunship"],                   //u      // uh. Does AREA work for this? Only lasts 5 minutes so maybe...
     ["SAM",           "TARGET", 1.0, 1.0,   0, 100, "u", ""],                            // balanced against ASF
-    ["CRUISEMISSILE", "AREA", 0.3, 0.1, 200,   100, "u", ""],
+    ["CRUISEMISSILE", "AREA", 0.4, 0.1, 150,   100, "u", ""],
     ["ORBITALSTRIKE",   "AREA", 0.2, 0.0, 300,   0, "f", ""],
-    ["UAV",           "TARGET", 1.0, 0.4,   0, 80,  "", "uavsAttack"]
+    ["UAV",           "TARGET", 0.6, 0.4,   0, 80,  "", "uavsAttack"]
```

Comment:
```suggestion
    ["UAV",           "TARGET", 0.3, 0.1,   0, 80,  "u", "uavsAttack"]
```
Given most UAV's are armed and have upwards of ~6 missiles, 1 support can technically be worth 3. They should be rare accordingly

It's also quite unfair, given they are pretty accurate with their ~6 missiles (though I guess that depends on modset)

Reply:
Unfair - no
Unfair is for something you cannot escape, you cannot destroy or it's reallyreally hard to do.

I think this will do
["UAV",           "TARGET", 0.4, 0.2,   0, 80,  "", "uavsAttack"]